src/icon/styles.scss

```diff
@@ -42,3 +42,9 @@
   inset-block-start: 0px;
   inset-inline-end: -3px;
 }
+
+@include styles.with-direction(
+  '.icon:is(.name-angle-left-double, .name-angle-left, .name-angle-right-double, .name-angle-right, .name-arrow-left, .name-caret-left-filled, .name-caret-right-filled, .name-audio-full, .name-audio-half, .name-audio-off, .name-external, .name-redo, .name-shrink, .name-undo, .name-view-vertical)'
+) {
```

Comment:
1. Need to have one more pass on `$selector` property. In our previous discussions I assumed it will be a short selector like `.icon`. But seeing this real use-case, I am not sure we want to go this way. To be discussed with other team members
2. We should make the 2nd `$direction` argument required too. Implicit `rtl` default is confusing

Reply:
1. `Icon` will be the ugliest by far. I would consider this an edge case. We could also simplify the `Icon` code having a single selector passed into the CSS (`icon.should-invert`) and do the computation in JavaScript instead of the long form expression of all of the icon names in the mixin.

2. Also fine by me.